xml/hardening.xml

```diff
@@ -2493,17 +2493,42 @@ Nmap run completed -- 1 IP address (1 host up) scanned in 221.669 seconds
      </screen>
 
     <para>
-     Note that running the <command>nmap</command> command can take quite a
-     while (in this example almost 4 minutes) depending on the options used. If
-     you remove the UDP port scan (leave out the <option>-U</option> option),
-     then <command>nmap</command> will finish the port scan nearly immediately.
-     The results of <command>nmap</command> can vary widely and might not show
+     Note that running the <command>nmap</command> command will only scan 
+     certain well-known ports by default. If you want to scan all ports you need to 
+     specify <command>-p 1-65535</command>. Doing this will slow down the scan
+     considerably. Especially for UDP scans the use of an asynchronous scanner
+     such as masscan 
+     <link xlink:href="https://software.opensuse.org/package/masscan"/>
+     is recommended.
```

Comment:
I think a </para> is missing here. Then it probably validates.

Reply:
the </para> is after the <screen> and for me it validates, strange